examples/demo/pingpong/src/pingpong/j/model/Racket.java

```diff
@@ -66,7 +66,7 @@ public void effect() {
 
 		@Override
 		public boolean guard() {
-			return Else();
+			return getTrigger(Ball.class).countdown <= 0;
```

Comment:
I have seen in the message of 4d8b652 that `else` cannot be exported. Does this mean that it hasn't been implemented yet, or do we face a theoretical problem? Either way, I suggest to add a comment here to remark that the original guard was `Else()`.